src/pineko/fonll.py

```diff
@@ -196,14 +225,24 @@ def produce_ptos(fns):
             base_pto,
             base_pto + 1,
         ]
-    else:
+    # plain FONLL with damping
+    elif is_mixed:
         return [base_pto for _ in range(7)]
+    # plain FONLL without damping
+    else:
+        return [base_pto for _ in range(5)]
 
 
-def produce_fonll_recipe(fns):
+def produce_fonll_recipe(fonll_fns, damp):
     """Produce the different theory cards according to which FONLL is asked for."""
     fonll_recipe = []
-    for fns, nfff, po, part in zip(FNS_LIST, NFFF_LIST, produce_ptos(fns), PARTS_LIST):
+    is_mixed = fonll_fns in MIXED_ORDER_FNS or damp != 0
```

Comment:
> Actually, for polarized we are using (right now) damping (since we don't have intrinsic), so are we blocking ourself here? right @Radonirinaunimi ?

DAMP should be working also with this PR. @felixhekhorn ( I mean it's not changed by this PR)

Reply:
It would not work when using the 5 theories but of course one can use the 7 theories instead

Reply:
yes and you will automatically generate 7 theories, so there is no problem.

Reply:
@felixhekhorn, just for confirmation, yes exactly! For polarized predictions, DAMP is **necessary** at NLO (especially for FONLL-A), otherwise the matching is not smooth and unstable close to the threshold.